config/environments/development.rb

```diff
@@ -13,12 +13,13 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
+  # Run rails dev:cache to toggle caching.
+  if Rails.root.join('tmp', 'caching-dev.txt').exist?
```

Comment:
[pronto-rubocop warning] Rails/FilePath: Please use `Rails.root.join('path/to')` instead.